packages/cli/src/index.js

```diff
@@ -12,4 +12,11 @@ config({
 })
 
 // eslint-disable-next-line no-unused-expressions
-yargs.commandDir('./commands').demandCommand().argv
+yargs
+  .commandDir('./commands')
+  .scriptName('rw')
```

Comment:
It looks like `.scriptName` should already be the name of the command

> https://github.com/yargs/yargs/blob/master/docs/api.md#scriptname0

I remember seeing a place where it wasn't, was it Windows?

Reply:
Oh, I see that you wanted that behaviour! My bad :)

Reply:
Yes, exactly, which is the reason for this PR. See #322 

I checked with the Yargs maintainer, and this was his recommendation.